central_v2/src/pages/Login.tsx

```diff
@@ -1,143 +1,219 @@
-import { styled } from '@mui/system';
-import { Link } from 'react-router-dom';
-import RightOnLogo from '../images/RightOnLogo.png';
+import React, { useState } from 'react';
+import { Box, TextField, Typography } from '@mui/material';
+import { useTheme, styled } from '@mui/material/styles';
 import { GoogleLogin } from '@react-oauth/google';
-import { IAuthAPIClient } from '@righton/networking';
+import { getUser, IAuthAPIClient } from '@righton/networking';
+import { useNavigate } from 'react-router-dom'; 
+import RightOnLogo from '../images/RightOnLogo.png';
+import { SignUpMainContainer } from '../lib/styledcomponents/SignUpStyledComponents';
+import CentralButton from "../components/button/Button";
+import { ButtonType } from '../components/button/ButtonModels';
+import ResetPassword from './ResetPassword';
+import { APIClientsContext } from '../lib/context/APIClientsContext';
+import { useTSAPIClientsContext } from '../hooks/context/useAPIClientsContext';
+
+const InnerBodyContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  // border: '1px solid blue',
+  flexDirection: 'column',
+  gap: '20px',
+  // height: '100vh',
+  // border: '1px solid red'
+  marginTop: '-8px',
+  marginBottom: '-100px'
+}));
+
+const UpperLogin = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  // border: '1px solid blue',
+  flexDirection: 'column',
+  alignItems: 'center',
+  gap: '20px',
+}));
+
+const UpperLoginText = styled(Typography)(({ theme }) => ({
+  fontFamily: 'Poppins, sans-serif',
+  fontWeight: 700, 
+  fontSize: '24px', 
+  color: '#02215F',
+  textAlign: 'center', 
+  // border: '1px solid green',
+
+}));
+
+const UpperLoginGoogleButton = styled(Typography)(({ theme }) => ({
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  color: '#0966E0', 
+  border: '2px solid #0966E0', // Set border to 2px with the same color
+  borderRadius: '8px', // Set border radius to 8px
+  backgroundColor: 'white', // Set background color to white
+  minHeight: '45.77px',
+  
+}));
+
+const OrText = styled(Typography)(({ theme }) => ({
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'center',
+  color: '#384466', 
+  // border: '1px solid yellow',
+  fontFamily: 'Poppins, sans-serif', 
+  fontWeight: 600, 
+  fontSize: '16px', 
+  // border: '1px solid green',
+
+}));
+
+const MiddleContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  // border: '1px solid green',
+  flexDirection: 'column',
+  gap: '12px'
+
+}));
+
+const ForgotPasswordButton = styled('button')(({ theme }) => ({
+  all: 'unset', // Resets all default styles for a button
+  color: '#02215F', 
+  fontFamily: 'Rubik, sans-serif', 
+  fontWeight: 400, 
+  fontSize: '16px', 
+  marginTop: '-8px',
+  cursor: 'pointer', // Add pointer cursor for button-like behavior
+  // border:'1px red solid'
+}));
+
+
+const UserTextField = styled(TextField)(({ theme }) => ({
+  border: '2px solid #CCCCCC', // Set border to 2px
```

Comment:
while I appreciate the comments, we can probably reduce the number here. Anything that is super self evident can be removed. I like the ones that describe the changes to the confusing mui css though (set placeholder text color etc)